pal/utils.ts

```diff
@@ -199,7 +201,7 @@ export function setTimeoutRAF (callback: (...args: any[]) => void, delay: number
     || window.oRequestAnimationFrame
     || window.msRequestAnimationFrame;
 
-    if (raf === undefined || globalThis.__globalXR?.isWebXR) {
+    if (EDITOR || raf === undefined || globalThis.__globalXR?.isWebXR) {
```

Comment:
Why use setTimeout in editor ?

Reply:
https://github.com/cocos/3d-tasks/issues/16221 raf cannot work normally when use `editor preview`, we guess it may be because it runs in the webview environment.